### PR TITLE
[ZEPPELIN-2764] update packages and documentation for ZeppelinHubRepo storage

### PR DESCRIPTION
**Pull request by khalidhuseynov**
### What is this PR for?
update the packages/documentation/links due to migration of domain and renaming

### What type of PR is it?
Documentation | Refactoring

### Todos
* [x] - update package/class names
* [x] - update config
* [x] - update docs

### What is the Jira issue?
[ZEPPELIN-2764](https://issues.apache.org/jira/browse/ZEPPELIN-2764)

### How should this be tested?
CI pass

### Questions:
* Does the licenses files need update? n
* Is there breaking changes for older versions? n
* Does this needs documentation? n

### CODE REVIEW

**Comment by jongyoul** [comment]
This is for fixing ASF rule. LGTM

File: zeppelin-server/src/main/java/org/apache/zeppelin/realm/ZeplRealm.java [hunk 1]
```diff
@@ -107,13 +107,13 @@ protected void onInit() {
...
+  public void setZeplUrl(String url) {
     if (StringUtils.isBlank(url)) {
-      LOG.warn("Zeppelinhub url is empty, setting up default url {}", DEFAULT_ZEPPELINHUB_URL);
-      zeppelinhubUrl = DEFAULT_ZEPPELINHUB_URL;
+      LOG.warn("Zepl url is empty, setting up default url {}", DEFAULT_ZEPPELINHUB_URL);
+      zeplUrl = DEFAULT_ZEPPELINHUB_URL;
     } else {
-      zeppelinhubUrl = (isZeppelinHubUrlValid(url) ? url : DEFAULT_ZEPPELINHUB_URL);
-      LOG.info("Setting up Zeppelinhub url to {}", zeppelinhubUrl);
+      zeplUrl = (isZeppelinHubUrlValid(url) ? url : DEFAULT_ZEPPELINHUB_URL);
```
Comment (felixcheung) [inline zeppelin-server/src/main/java/org/apache/zeppelin/realm/ZeplRealm.java]: DEFAULT_ZEPPELINHUB_URL should change?

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/repo/zepl/ZeppelinHubRepoTest.java [hunk 2]
```diff
@@ -55,73 +56,73 @@ private ZeppelinhubRestApiHandler getMockedZeppelinHandler() throws HttpExceptio
...
 
   @Test
   public void testGetZeppelinHubWsEndpoint() {
-    System.setProperty(ZeppelinHubRepo.ZEPPELIN_CONF_PROP_NAME_SERVER, testAddr);
+    System.setProperty(ZeplRepo.ZEPPELIN_CONF_PROP_NAME_SERVER, testAddr);
 
     ZeppelinConfiguration config = new ZeppelinConfiguration();
-    ZeppelinHubRepo repository = new ZeppelinHubRepo(config);
+    ZeplRepo repository = new ZeplRepo(config);
     assertThat(repository.getZeppelinhubWebsocketUri(config)).isEqualTo("ws://zeppelinhub.ltd:80/async");
```
Comment (felixcheung) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/repo/zepl/ZeppelinHubRepoTest.java]: ws://zeppelinhub.ltd:80/async should change?

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/repo/zepl/ZeppelinHubRepoTest.java [hunk 3]
```diff
@@ -55,73 +56,73 @@ private ZeppelinhubRestApiHandler getMockedZeppelinHandler() throws HttpExceptio
...
 
-    System.setProperty(ZeppelinHubRepo.ZEPPELIN_CONF_PROP_NAME_SERVER, "https://zeppelinhub.ltd");
+    System.setProperty(ZeplRepo.ZEPPELIN_CONF_PROP_NAME_SERVER, "https://zeppelinhub.ltd");
```
Comment (felixcheung) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/repo/zepl/ZeppelinHubRepoTest.java]: and https://zeppelinhub.ltd"?

**Pull request closed without merging**